AFQ/definitions/mask.py

```diff
@@ -72,28 +72,42 @@ class MaskFile(Definition):
 
     Parameters
     ----------
-    suffix : str
+    path : str, optional
+        path to file to get mask from. Use this or suffix.
+        Default: None
+    suffix : str, optional
         suffix to pass to bids_layout.get() to identify the file.
-    filters : str
+        Default: None
+    filters : str, optional
         Additional filters to pass to bids_layout.get() to identify
         the file.
         Default: {}
 
     Examples
     --------
     seed_mask = MaskFile(
-        "WM_mask",
-        {"scope":"dmriprep"})
-    api.AFQ(tracking_params={"seed_mask": seed_mask,
+        suffix="WM_mask",
```

Comment:
Does this actually work? I've run into some issues where pybids only considered something to be a suffix if it was the single contiguous string after the last underscore before the extension.